### PR TITLE
Added dropdowns and colorpickers to the SceneEffectCategory stuff.

### PR DESCRIPTION
**Pull request by krypto5863**
Threw these together real quick, I only tested in HS2 and KKS, so lemme know if fixes or changes need to be made.

### CODE REVIEW

**Review by ManlyMarco — CHANGES_REQUESTED** [review]
The changes look good, but whitespace got messed up with random tabs. Make sure your editor is set to using spaces for indents.

**Comment by krypto5863** [comment]
Should be fixed now, I'm a tab enjoyer. Might add a few more controls to this later.

**Comment by ManlyMarco** [comment]
Do you want me to wait with reviewing until you finish adding them? You can mark the PR as a draft in that case.

**Comment by krypto5863** [comment]
Nah should be good to go now. Think I've done most of the simple stuff with this already.

File: src/Shared.CharaStudio/Studio/UI/SceneEffectsCategory.cs [hunk 1]
```diff
@@ -68,6 +78,12 @@ private static void Initialize()
 #if !PH
             _inputSource = sef.transform.Find(inputSourcePath)?.gameObject ?? throw new ArgumentException("Could not find " + inputSourcePath);
             _buttonSource = sef.transform.Find(buttonSourcePath)?.gameObject ?? throw new ArgumentException("Could not find " + buttonSourcePath);
+            _dropdownSource = sef.transform.Find(dropdownSourcePath)?.gameObject ?? throw new ArgumentException("Could not find " + dropdownSourcePath);
+            _colorPickerSource = sef.transform.Find(colorPickerSourcePath)?.gameObject ?? throw new ArgumentException("Could not find " + colorPickerSourcePath);
+
+            //Not too proud of these, but they work fine and should continue to work fine.
+            _expandSprite = Resources.FindObjectsOfTypeAll<Sprite>().FirstOrDefault(r => r.name.Equals("sp_sn_09_00_05"));
+            _collapseSprite = Resources.FindObjectsOfTypeAll<Sprite>().FirstOrDefault(r => r.name.Equals("sp_sn_09_00_03"));
```
Comment (ManlyMarco) [inline src/Shared.CharaStudio/Studio/UI/SceneEffectsCategory.cs:86]: Why not do this?
```suggestion
            var allsprites = Resources.FindObjectsOfTypeAll<Sprite>();
            _expandSprite = allSprites.FirstOrDefault(r => r.name.Equals("sp_sn_09_00_05"));
            _collapseSprite = allSprites.FirstOrDefault(r => r.name.Equals("sp_sn_09_00_03"));
```

File: src/Shared.CharaStudio/Studio/UI/SceneEffectsCategory.cs [hunk 2]
```diff
@@ -68,6 +78,13 @@ private static void Initialize()
 #if !PH
             _inputSource = sef.transform.Find(inputSourcePath)?.gameObject ?? throw new ArgumentException("Could not find " + inputSourcePath);
             _buttonSource = sef.transform.Find(buttonSourcePath)?.gameObject ?? throw new ArgumentException("Could not find " + buttonSourcePath);
+            _dropdownSource = sef.transform.Find(dropdownSourcePath)?.gameObject ?? throw new ArgumentException("Could not find " + dropdownSourcePath);
+            _colorPickerSource = sef.transform.Find(colorPickerSourcePath)?.gameObject ?? throw new ArgumentException("Could not find " + colorPickerSourcePath);
+
+            //Not too proud of these, but they work fine and should continue to work fine.
+            var allsprites = Resources.FindObjectsOfTypeAll<Sprite>();
```
Comment (ManlyMarco) [inline src/Shared.CharaStudio/Studio/UI/SceneEffectsCategory.cs:85]:
```suggestion
            var allSprites = Resources.FindObjectsOfTypeAll<Sprite>();
```

**Review by ManlyMarco — APPROVED** [review]

**Pull request merged**